modules/tests/test/dataobject/operations.go

```diff
@@ -5,17 +5,27 @@ import (
 	"time"
 
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/constants"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
 	cdiv1beta1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 	cdicliv1beta1 "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned/typed/core/v1beta1"
 )
 
-func WaitForSuccessfulDataVolume(cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace, name string, timeout time.Duration) error {
+func WaitForSuccessfulDataVolume(k8sClient *kubernetes.Clientset, cdiClientSet cdicliv1beta1.CdiV1beta1Interface, namespace, name string, timeout time.Duration) error {
```

Comment:
Can't the `kubecli.KubevirtClient` be used for everything? It implements `kubernetes.Clientset` and provides the CDI client with method `CdiClient()`.